# *Structural variation analysis pipeline for WGS of cancer genomes *

## Description
This workflow will run the [SvABA](https://github.com/walaj/svaba) structural variation (SV) analysis for a set of tumor-normal pairs, starting from the aligned BAM files. The analysis includes structural variant prediction and and determine SV classes. SV classes are predicted by combining SV results with copy number results previously generated by [TitanCNA](https://github.com/gavinha/TitanCNA).

## Contact
Gavin Ha  
Fred Hutchinson Cancer Research Center  
contact: <gavinha@gmail.com> or <gha@fredhutch.org>  
Date: January 2, 2019

## Requirements
### Software packages or libraries
 - R-3.4
	-optparse  
	-data.table  
	-GenomicRanges  
	-GenomeInfoDb  
	-VariantAnnotation  
	-plyr  
	-ggplot2  
	-reshape2  
	-stringr  
 - Python 3.4 
   - snakemake-5.4
 - [samtools-1.3.1](http://www.htslib.org/)

